www/src/examples/Table/Responsive.js

```diff
@@ -2,41 +2,37 @@
   <thead>
     <tr>
       <th>#</th>
-      <th>Table heading</th>
-      <th>Table heading</th>
-      <th>Table heading</th>
-      <th>Table heading</th>
-      <th>Table heading</th>
-      <th>Table heading</th>
+      {(function () {
```

Comment:
```suggestion
      {Array.from({length:5}).map(_ => <th>Table heading</th>)}
```
should work too for both cases.

Reply:
@mxschmitt  Thanks ! 👍